subsys/caf/modules/ble_adv.c

```diff
@@ -583,7 +583,6 @@ static bool handle_ble_peer_operation_event(const struct ble_peer_operation_even
 		break;
 
 	case PEER_OPERATION_ERASED:
-		__ASSERT_NO_MSG(cur_identity == event->bt_stack_id);
```

Comment:
Why this check was removed ?

Reply:
because when we erase dongle peer, cur_identity  is still the one used without a dongle so different then the one from event.

Reply:
Discussed offline, found a bug. It was fixed during discussion.